packages/visualizations/src/components/Format/index.ts

```diff
@@ -1,30 +1,20 @@
 import BooleanFormat from './BooleanFormat.svelte';
 import DateFormat from './DateFormat.svelte';
 import GeoFormat from './GeoFormat.svelte';
-import LongTextFormat from './LongTextFormat.svelte';
 import NumberFormat from './NumberFormat.svelte';
-import ShortTextFormat from './ShortTextFormat.svelte';
+import TextFormat from './TextFormat.svelte';
 import URLFormat from './URLFormat.svelte';
-import { isValidRawValue } from './utils';
+import { isValidValue } from './utils';
 
 const Format = {
     boolean: BooleanFormat,
```

Comment:
This Format constant doesn't seem to be used somewhere. 
Do we need it in our application ?